### PR TITLE
no slow isdc query

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

File: cdci_osa_plugin/osa_dataserve_dispatcher.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def __init__(self,
         self.assume = assume
         self.inject = inject
 
-        self._test_products_with_astroquery = True
+        self._test_products_with_astroquery = False # TODO: this should be configurable
```
Comment (burnout87) [inline cdci_osa_plugin/osa_dataserve_dispatcher.py:129]: What is the main of having this parameter?

**Review by burnout87 — COMMENTED** [review]
just a question

File: cdci_osa_plugin/osa_dataserve_dispatcher.py (hunk 1 above)
Comment (volodymyrss) [reply cdci_osa_plugin/osa_dataserve_dispatcher.py:129]: the main purpose? It decides if test_products are derived with astroquery - as used below.

**Review by burnout87 — APPROVED** [review]

**Comment by volodymyrss** [comment]
I realized I could improve that part a bit, a moment.

**Pull request merged**